ui/partials/perf/comparerror.html

```diff
@@ -0,0 +1,6 @@
+<div class="alert alert-danger" role="alert">
+    <p ng-repeat="error in errors">
+        <span class="fa fa-warning"></span> <span class="revision-holder">{{error}}</span>
```

Comment:
> I think this is probably more trouble than it's worth at this point

Agreed :) @MikeLing clarified in channel he was manually removing a character from the SHA instead of changing a character, to trigger a failure condition for the screen grabs, but I thought they were truly varying lengths.